cognite_toolkit/_cdf_tk/commands/clean.py

```diff
@@ -85,7 +85,7 @@ def clean_resources(
         if nr_of_items == 0:
             return ResourceDeployResult(name=loader.display_name)
 
-        existing_resources = loader.retrieve(loader.get_ids(loaded_resources)).as_write()
```

Comment:
This cause the Hosted extractor source to fail as it does not support `.as_write`.
![image](https://github.com/user-attachments/assets/9d14b613-58e2-412c-885e-6e6c77c165e1)

However, looking through the code, `existing_resources` is only used to look up IDs, and that we might as well do from the read/response items instead of the write/request items.